clang/include/clang/AST/Expr.h

```diff
@@ -306,8 +307,9 @@ class Expr : public ValueStmt {
     MLV_NotObjectType,
     MLV_IncompleteVoidType,
     MLV_DuplicateVectorComponents,
+    MLV_DuplicateMatrixComponents,
     MLV_InvalidExpression,
-    MLV_LValueCast,           // Specialized form of MLV_InvalidExpression.
+    MLV_LValueCast, // Specialized form of MLV_InvalidExpression.
```

Comment:
This file has inconsistent whitespaces changes not related to the PR. The whitespace here before the comment is removed, and on lines 345-355 similar whitespaces are added. These whitespace-only changes should be reverted.